nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/PutMongo.java

```diff
@@ -155,12 +155,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
         try {
             // Read the contents of the FlowFile into a byte array
             final byte[] content = new byte[(int) flowFile.getSize()];
-            session.read(flowFile, new InputStreamCallback() {
-                @Override
-                public void process(final InputStream in) throws IOException {
-                    StreamUtils.fillBuffer(in, content, true);
-                }
-            });
+            session.read(flowFile, in -> StreamUtils.fillBuffer(in, content, true));
```

Comment:
this is causing the error on Ci because of the unused import which introduces. Since this is not related with the PR, can we get it back to its original state?